test_sets/PTU_no_certificate_exists_at_module_config.txt

```diff
@@ -0,0 +1,57 @@
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/018_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/019_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_ON_RPC_encrypted_true.lua Check_that_SDL_will_return_RPC_NACK_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/020_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/021_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Audio_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/022_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Video_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/023_ATF_P_TC_navi_app_PTU_no_certificate_ForceProtectedService_OFF_Hybrid_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/024_ATF_P_TC_media_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/025_ATF_P_TC_comm_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/026_ATF_P_TC_default_app_PTU_no_certificate_ForceProtectedService_OFF_RPC_encrypted_true.lua Check_that_SDL_will_return_StartService_ACK_encrypt_false_when_PTU_brings_no_certificate
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/027_ATF_P_TC_navi_app_PTU_fail_ForceProtectedService_OFF_Audio_encrypted_true.lua Check_that_SDL_will_return_Audio_ACK_encrypt_false_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/028_ATF_P_TC_navi_app_PTU_fail_ForceProtectedService_OFF_Video_encrypted_true.lua Check_that_SDL_will_return_Video_ACK_encrypt_false_when_PTU_fail
+./test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/029_ATF_P_TC_navi_app_PTU_fail_ForceProtectedService_OFF_Hybrid_encrypted_true.lua Check_that_SDL_will_return_Hybrid_ACK_encrypt_false_when_PTU_fail
```

Comment:
@istoimenova As I know Hybrid service (15) available on app registering. Do not need to start this service. Also it should be secure if RPC service becomes secure. Please clarify am I correctly remember.